kitsune/wiki/tests/test_templates.py

```diff
@@ -380,6 +380,35 @@ def test_links_follow(self):
         doc = pq(response.content)
         assert 'rel' not in doc('#doc-content a')[0].attrib
 
+    def test_document_with_fallback_locale(self):
+        """The document template falls back to fallback locale if there is
+        custom wiki fallback mapping for the locale and the locale have no translation
+        exists."""
+        # Create a English Document and a pt-BR Document
+        en_rev = revision(save=True, content='Some text.',
+                          is_approved=True, is_ready_for_localization=True)
+        trans_doc = document(parent=en_rev.document, locale='pt-BR', save=True)
+        trans_rev = revision(document=trans_doc, save=True,
+                             content='Translated text.', is_approved=True)
+        # Make the revision as current revision of the document
+        trans_doc.current_revision = trans_rev
+        trans_doc.save()
+
+        # Now get the pt-PT version of the document. As pt-PT has a fallback locale pt-BR
```

Comment:
# Get the pt-PT version of the document.
# Resolve to the pt-BR version
# because pt-PT has pt-BR set in FALLBACK_LOCALES in wiki/config.py